Source/SelfService/Web/screens/applicationsScreen/applicationsList.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import React from 'react';
+
+import { ShortInfoWithEnvironment } from '../../api/api';
+
+import { Button, List } from '@mui/material';
+
+const styles = {
+    list: {
+        p: 0,
+        mt: 2.5,
+        display: 'inline-block'
+    },
+    button: {
+        display: 'block',
+        inlineSize: '100%',
+        minInlineSize: '9.6875rem',
+        minBlockSize: '2.25rem',
+        letterSpacing: '0.06em',
+        typography: 'body2',
+        fontWeight: 500,
```

Comment:
This is just missing a little space in between the buttons:
```suggestion
        fontWeight: 500,
        mb: 2,
```